GitUI/CommandsDialogs/BrowseDialog/FormBrowseMenus.cs

```diff
@@ -115,6 +132,122 @@ static ToolStripItem CreateItem(ToolStrip senderToolStrip)
             }
         }
 
+        private static void CreateToolStripSubMenus(ToolStrip senderToolStrip, ToolStripMenuItem toolStripItem)
+        {
+            const string toolbarPrefix = "formbrowse_toolbar_visibility_";
```

Comment:
This is greatly bloat the settings 🤔 
I wonder whether we need to store this information in the appsettings, or store it separately like we store windows positions (`WindowPositionList`)?

Reply:
Using the appsettings is very convenient so I prefer to continue to do this way.
I changed a little the way to save the value so now only when an item is hidden, a setting with the value `false` is added to the settings. Otherwise nothing is store.
So a lot less bloat.